win32ss/user/user32_vista/dpi.c

```diff
@@ -11,10 +11,15 @@
 #include <windef.h>
 #include <wingdi.h>
```

Comment:
```suggestion
#include <winbase.h>
#include <wingdi.h>
```
Weird that winbase.h is needed.